<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaRay::SceneInspector</class>
 <widget class="QWidget" name="GammaRay::SceneInspector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QComboBox" name="sceneComboBox"/>
        </item>
        <item>
         <widget class="QSplitter" name="splitter_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QSplitter" name="splitter_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QWidget" name="layoutWidget_6">
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="KFilterProxySearchLine" name="screneTreeSearchLine"/>
             </item>
             <item>
              <widget class="QTreeView" name="sceneTreeView">
               <attribute name="headerShowSortIndicator" stdset="0">
                <bool>true</bool>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="GammaRay::GraphicsSceneView" name="graphicsSceneView" native="true"/>
          </widget>
          <widget class="GammaRay::PropertyWidget" name="scenePropertyWidget" native="true"/>
         </widget>
        </item>
       </layout>
 </widget>
 <resources/>
 <connections/>
 <customwidgets>
  <customwidget>
   <class>KFilterProxySearchLine</class>
   <extends>QWidget</extends>
   <header location="global">kde/kfilterproxysearchline.h</header>
  </customwidget>
  <customwidget>
   <class>GammaRay::PropertyWidget</class>
   <extends>QWidget</extends>
   <header location="global">propertywidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GammaRay::GraphicsSceneView</class>
   <extends>QWidget</extends>
   <header location="global">tools/sceneinspector/graphicssceneview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
</ui>
